src/pytest_plugins/filler/ported_tests.py

```diff
@@ -0,0 +1,161 @@
+"""
+A pytest plugin that shows `ported_from` marker information.
+
+This plugin extracts and displays information from @pytest.mark.ported_from markers,
+showing either the static filler file paths or associated PR URLs.
+
+Usage:
+------
+# Show static filler file paths
+uv run fill --show-ported-from tests/
+
+# Show PR URLs instead
+uv run fill --show-ported-from=prs tests/
+
+The plugin will:
+1. Collect all test items with @pytest.mark.ported_from markers
+2. Extract either the file paths (first positional argument) or PR URLs (pr keyword argument)
+3. Output a deduplicated, sorted list, one per line
+4. Skip test execution (collection only)
+
+Marker Format:
+--------------
+@pytest.mark.ported_from(
+    ["path/to/static_filler1.json", "path/to/static_filler2.json"],
+    pr=[
+        "https://github.com/ethereum/execution-spec-tests/pull/1234",
+        "https://github.com/ethereum/execution-spec-tests/pull/5678",
+    ],
+)
+"""
+
+import re
+from typing import Set
+from urllib.parse import urlparse
+
+import pytest
+
+
+def convert_to_filled(file_path: str) -> str | None:
+    """Convert source link to filler to filled test path."""
+    path = urlparse(file_path).path
+    if "/src/" in path:
+        path = path.split("/src/", 1)[1]
+
+    if path.endswith((".sh", ".js")):
+        return None
+
+    # Remove "Filler" from the path components
+    path = path.replace("TestsFiller", "Tests")
+
+    # Replace file extension to .json
+    path = re.sub(r"Filler\.(yml|yaml|json)$", r".json", path)
+
+    return path
+
+
+def pytest_addoption(parser: pytest.Parser):
+    """Add command-line options to pytest."""
+    ported_from_group = parser.getgroup(
+        "ported_from", "Arguments for showing ported_from marker information"
+    )
+    ported_from_group.addoption(
+        "--show-ported-from",
+        action="store",
+        dest="show_ported_from",
```

Comment:
Can you add this and `links_as_filled` to the `pytest_plugins.help` plugin? Then they'll get printed as a dedicated group when you run `uv run fill --help`.

Reply:
how? it is not displayed by default. as well as fill static tests

Reply:
ok that was a bit tricky. that instead of identificator there a substring match.